README.md

```diff
@@ -221,6 +221,26 @@ The documentation is generated with
 $ make doc
 ```
 
+#### macOS notes
+
+## Openssl headers
+
+Newer versions of macOS don't include the openssl headers by purpose. You'll need to install
+them via `brew install openssl` or manually.
+
+Homebrew openssl headers however don't get symlinked in /usr/local, so you need to tell
+luarocks where to find them via `OPENSSL_DIR`.
+
+```
+OPENSSL_DIR=/usr/local/opt/openssl/ FLAGS='--local' make busted
+```
```

Comment:
Ditto, I don't think this is a good place to teach users how to properly install OpenSSL on macOS. Makes for opinionated setups that this library should not concern itself with...

Reply:
I can remove the reference to macOS, which actually is not strictly relevant since it looks like this problem also happen on some linux distribution.

I just wanted to let developers know that the Makefile already supports this and they only need to set an env variable to fix it. Same for your next comment on `--local`.